documentation/library_guide/kernel_templates/single_pass_scan.rst

```diff
@@ -78,7 +78,7 @@ Parameters
 
   Current limitations:
 
-  - The function will internally block until the issued kernels have completed execution.
+  - Except for the case described in the :ref:`single-work-group note <scan-single-wg>`, the function will internally block until the issued kernels have completed execution.
```

Comment:
Have you tested to see if this cross link works? I ask because it didn't work for me. The issue might be the underscore on line 168 _scan-single-wg, where on line 81 it is referenced as scan-single-wg. 

The sentence LGTM otherwise.

Reply:
Yes, in the generated HTML, it does create a link. When clicked on, it scrolls down to the correct note.
![image](https://github.com/user-attachments/assets/94a5f141-3ece-4e4b-b3b6-a845a973b567)

According to the [reStructuredText documentation](https://www.sphinx-doc.org/en/master/usage/referencing.html#ref-role) on cross-referencing, they state the following:

> Reference labels must start with an underscore. When referencing a label, the underscore must be omitted.

So I believe the underscore is necessary in the label but not in the reference to the label.

Reply:
Thank you for the additional information! I approve.